services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+
+
+User = get_user_model()
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str = "",
+    last_name: str = ""
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+    return user
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        return User.objects.get(id=user_id)
+    except ObjectDoesNotExist:
+        return None

```

Comment:
if User doesn't exist - let the error happen, it would make sense to stop program execution here instead of dealing with None values